### PR TITLE
PLAT-129761: Added "children" for Spinner

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Issues: 
agate/Spinner was displaying only an icon, although the inner ui/Spinner supports also content text

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
agate/Spinner can display a text together with the corresponding icon

### Links
[//]: # (Related issues, references)
PLAT-129761

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Review by stanca-pop-lgp — APPROVED** [review]

File: samples/sampler/stories/qa/Spinner.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		() => (
+			<div>
+				<div
+					style={{
+						height: ri.scaleToRem(300),
+						border: ri.scaleToRem(6) + ' dotted blue'
+					}}
+				>
+					<SkinnedSpinner
+						transparent={boolean('transparent', Spinner, false)}
```
Comment (0x64) [inline samples/sampler/stories/qa/Spinner.js:45]: The order of knob props is different from the default sampler which has alphabetically ordered props. The order could be not an alphabetical order if intended, but it looks better to have the same order between the default and QA samplers.

**Review by 0x64 — APPROVED** [review]
:lgtm:

File: samples/sampler/stories/qa/Spinner.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+storiesOf('Spinner', module)
+	.add(
+		'with long content',
+		() => (
+			<div>
+				<div
+					style={{
+						height: ri.scaleToRem(300),
+						border: ri.scaleToRem(6) + ' dotted blue'
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Spinner.js:41]: It's miscellaneous but, I'll make some changes to this and merge. "6" seems too thick.

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**